docs/src/tutorials/diffusion_implicit_heat_equation.md

```diff
@@ -0,0 +1,186 @@
+# Solving the heat equation with diffusion-implicit time-stepping.
```

Comment:
```suggestion
# IMEX Integrators
```

This title is more informative as to what the tutorial is actually trying to teach.

Reply:
If it's moved, the original title makes sense though.